src/backend/src/prisma/schema.prisma

```diff
@@ -636,3 +653,42 @@ model Meeting {
   team         Team?  @relation(fields: [teamId], references: [teamId])
   teamId       String
 }
+
+model Wbs_Proposed_Changes {
+  proposedChangeId     String                         @id @default(uuid())
+  name                 String
+  status               WBS_Element_Status
+  projectLeadId        Int
+  projectManagerId     Int
+  deletedByUserId      Int
+  links                Link[]                         @relation(name: "proposedChangeLinks")
+  changRequestId       Int                            @unique
```

Comment:
this should have a one to one relation with a change request

Reply:
should the wbsproposed change in the change request model be optional?